src/app/examples/form/components/form.component.spec.ts

```diff
@@ -1,35 +1,104 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { Store } from '@ngrx/store';
+import { createComponent, RenderResult } from 'ngx-testing-library';
 
 import { MockStore, TestingModule } from '@testing/utils';
 
+import { NotificationService } from '@app/core/notifications/notification.service';
 import { State } from '../../examples.state';
 import { FormState } from '../form.model';
-import { initialState } from '../form.reducer';
 import { FormComponent } from './form.component';
-import { NotificationService } from '@app/core/notifications/notification.service';
+import { initialState } from '../form.reducer';
+import { ActionFormUpdate } from '../form.actions';
 
-describe('FormComponent', () => {
-  let component: FormComponent;
-  let fixture: ComponentFixture<FormComponent>;
+fdescribe('FormComponent', () => {
   let store: MockStore<State>;
+  let renderResult: RenderResult;
+  let dispatchSpy: jasmine.Spy;
 
-  beforeEach(async(() => {
-    TestBed.configureTestingModule({
-      imports: [TestingModule],
-      declarations: [FormComponent],
-      providers: [NotificationService]
-    }).compileComponents();
+  beforeEach(async () => {
+    renderResult = await createComponent<FormComponent>(
```

Comment:
Maybe call this `renderedComponent` ? So then `  renderedComponent.click`, or just rendered ? But I understand that the type is `RenderedResult`, just thinking if I would understand what it means ?

Reply:
What about just `component`?
At the time I wrote this there was another variable named `component`, thus I could not use it...

Reply:
Yeah that would make sense too ;)